contracts/lib/Destructable.sol

```diff
@@ -12,6 +12,6 @@ import "./Ownable.sol";
 contract Destructable is Ownable {
 
     function destruct() external onlyOwner {
-        selfdestruct(owner);
+        selfdestruct(address(uint160(owner)));
```

Comment:
Does `address(owner)` not work?

Reply:
No. I could make the owner payable in `Ownable` but since it is not required there, I would rather not.